src/sites/image/xxx.fopkodiak.site.js

```diff
@@ -0,0 +1,22 @@
+_.register({
+  rule: {
+    host: /^xxx\.fopkodiak\.site$/,
+    path: /img-.*/,
```

Comment:
```suggestion
    path: /^\/img-/,
```
Probably always starts with `/img-`. Since we don't need the rest part, just leave it alone.

Reply:
PR updated.